### PR TITLE
Modify CHANGELOG and VERSION for release v0.2.3

### PR DESCRIPTION
**Pull request by zyiou**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.  The format
 Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+## 0.2.3 10-30-2020
+### Changed
+- Change `DateTimeSeconds` type for information elements to `uint32` following RFC7011. (#59, @zyiou)
+- Change PEN number for Antrea to '56506' (assigned by IANA). (#60, @zyiou)
+### Added
+- Improve test coverage for IPv6 support. (#61, @zyiou)
```
Comment (srikartati) [inline CHANGELOG.md:12]: Could you please add about the support for IPv6 clusterIP in this patch release?

Comment (zyiou) [reply CHANGELOG.md]: Updated. Thanks!

**Review by srikartati — APPROVED** [review]
LGTM.

**Review by srikartati — DISMISSED** [review]

**Comment by zyiou** [comment]
@srikartati I removed the CHANGELOG about PR#61, please have a quick review again. Thanks!

**Review by srikartati — APPROVED** [review]

**Pull request merged**